__tests__/app.test.js

```diff
@@ -116,11 +116,48 @@ describe("GET/api/categories", () => {
   .expect(200)
   .then((response)=>{
     
-    console.log(response.body, 'inside test')
     expect(response.body.comments).toStrictEqual([])
   })
  })
 
+ describe('POST api/reviews/:review_id/comments',()=>{
+    test('400: not found', ()=>{
+      return request(app)
+      .post('/api/reviews/40/comments')
+      .expect(404)
+      .then((response)=>{
+        const msg = response.body.msg
+        expect(msg).toBe("not found")
+      })
+    })
+    test('400:invalid review_id',()=>{
+      return request(app)
+      .get('/api/reviews/orange/comments')
+      .expect(400)
+      .then((response)=>{
+        const msg = response.body.msg
+        expect(msg).toBe("bad request")
+      })
+ })
```

Comment:
You are missing 2 sad path tests. 

400 - What if I send the body without the correct keys? Eg. { notusername: "philippaclaire9", information: "some text"} 

404 - What if i send a post request to /api/reviews/3/comments but the username doesn't exist in the database? Eg. { username: "Rob", body: "Here is my comment"}